app/scripts/views/sign_up.js

```diff
@@ -157,15 +157,17 @@ define(function (require, exports, module) {
         chooseWhatToSyncCheckbox: this.broker.hasCapability('chooseWhatToSyncCheckbox'),
         email: prefillEmail,
         error: this.error,
+        isAmoMigration: this.isAmoMigration(),
         isCustomizeSyncChecked: relier.isCustomizeSyncChecked(),
         isEmailOptInVisible: this._isEmailOptInEnabled(),
-        isMigration: this.isMigration(),
         isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
         isSync: isSync,
+        isSyncMigration: this.isSyncMigration(),
         password: prefillPassword,
         serviceName: relier.get('serviceName'),
         shouldFocusEmail: autofocusEl === 'email',
-        shouldFocusPassword: autofocusEl === 'password'
+        shouldFocusPassword: autofocusEl === 'password',
+        signinUri: '/signin'
```

Comment:
The URL has to be customized depending on the flow, we do a bit of magic like so:

```js
signinUri: this.broker.transformLink('/signin')
```

For the OAuth flow, it'll use `/oauth/signin`, for the non-OAuth flow, it'll use `/signin`